transcom/mymove
# B19228 MAIN 20240618 - Fix sidebar text spilling paste the page on MTO page

## INT PR
[13104](https://github.com/transcom/mymove/pull/13104)

## Agility ticket
[B19228](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19228)

## Summary
When an office user was on the MTO page and the sidebar text (e.g. facility info and address) was very long, it would spill paste the end of the page.  Fix was made to wrap the text when it exceeded the sidebar width.

### How to test
* Create a new move
  - Enter customer info
  - Enter orders info
  - Create shipment: NTS-Release 
* SC:
  - Review and edit orders
    * Update orders fields like normal
    * Under NTS accounting codes, enter the following value for SAC
      - VQTLP02819JNDLT73405BRWEM91062XZCAF45870SYOHG16321KIMNP80943
  - Edit shipment
    * Enter the following values:
      - Facility Name: AAA Ace Number One Best Ever Climate Controlled Storage
      - Address 1:  12076 Whispering Oaks Lush Green Meadow Lane
      - City: Rancho Santa Margarita
      - Service order number: T782143905612
  - Submit move
* TOO:  
  - Edit Shipment details
    * Set weight to: 2000
    * Save
  - Approve shipment
  - After approving, page will change to MTO page.
  - Scroll down to the side bar and confirm text is not going past the end of the page.

## Screenshots
Before:
![image](https://github.com/transcom/mymove/assets/146897935/3058c21f-e988-45fc-a7cd-8999b33abd34)

After:
![image](https://github.com/transcom/mymove/assets/146897935/ca02f6e5-6d5a-41aa-b257-99bebbaeb5ed)
